modules/ROOT/nav.adoc

```diff
@@ -10,6 +10,8 @@ include::partial$nav-migration.adoc[]
  ** xref:global-settings-configuration.adoc[Global Settings Configuration]
 * xref:mule-application-about.adoc[Mule Apps]
  ** xref:about-flows.adoc[Flows and Subflows]
+ ** xref:understanding-orchestration-using-mule.adoc[Understanding Orchestration Using Mule]
```

Comment:
Pablo LaGreca mentioned this on the issue to migrate this doc: Put under Orchestration in the TOC defined by https://www.mulesoft.org/jira/browse/MULE-16173

Reply:
We are trying to create a TOC that does not have as many levels as the one Pablo is suggesting. Specifically, we are trying to keep it to two levels. So for now, I am putting the orchestration topics directly under  MULE Apps. We plan to revisit the TOC with Pablo once we have more topics in it